README.md

```
@@ -160,19 +160,59 @@ asyncio.run(main())
 
 ### Error Handling
 
+Sia-scraper provides a typed exception hierarchy with two independent trees:
+
+**Rust exceptions** (from `sia_scraper_rust`, re-exported via `sia_scraper.core.exceptions`):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)